env/file_system_tracer.h

```diff
@@ -72,9 +72,11 @@ class FileSystemPtr {
         fs_tracer_(
             std::make_shared<FileSystemTracingWrapper>(fs_, io_tracer_)) {}
 
-  explicit FileSystemPtr(std::shared_ptr<FileSystem> fs)
```

Comment:
This class has the same questions.  Why do we need it?  Can we change the DB to store either the "raw" FileSystem" or a "TracedFileSystem" based on what the input IO settings would be?